modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -127,6 +128,12 @@ The following components are affected:
 </ExternalHyperlink>
 ```
 
+## Loading Sparkles
+
+**PR:** [#3120](https://github.com/Workday/canvas-kit/pull/3120)
+
+It now uses new styling utilities and CSS tokens.
+
```

Comment:
So we don't need this section for our docs as the rest of the PRs are on line 95. 

You can inline the PR number and the link on that line.